Entities/AtomGroup.cpp

```diff
@@ -1,4 +1,6 @@
 #include "AtomGroup.h"
+
+#include "Actor.h"
```

Comment:
Yeah, I hate the billion arguments. In my opinion adding a new parameter is 

A. Still coupling, just now implicit instead of explicit 
B. More difficult to read as the logic is now indirect instead of directly stated inline at the point where it matters

However I very much agree that the LimbPath knowing about actor/controller is *also* awful. Ideally, it’d be a property of the LimbPath instead, “IntendedMovementDirection”, but alas.

Reply:
Yeah, this may be a case where it'd be good to have some data struct instead of 80 million args.

And you're right that it's coupled either way. This whole method is kinda mess of that regardless. Anyway, feel free to resolve, I just wanted to mention this.